DTS-STN/Service-Canada-Labs
# Project page unit tests

# [Create Unit Tests for Pages](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=217872)

Updated mockStore.js data to include current projects, also removed data regarding old projects.
Created unit tests for each of the project pages.

## Test Instructions

1. yarn dev (ensure each page works correctly)
2. yarn test:unit (ensure all tests pass successfully)
